### PR TITLE
modify mount topics conditionalization for cloud

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/<add-your-issue-number-here>
Review deadline:

## Page previews

https://deploy-preview-894--redpanda-docs-preview.netlify.app/current/manage/mountable-topics/

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by Deflaimun** [comment]
@micheleRP should we remove this line for cloud? 
https://github.com/redpanda-data/docs/blob/0a3e3c8f9f83c9307c974d9d54cc7e18ad158de5/modules/manage/partials/mountable-topics.adoc?plain=1#L17

**Comment by kbatuigas** [comment]
> @micheleRP should we remove this line for cloud? 
> https://github.com/redpanda-data/docs/blob/0a3e3c8f9f83c9307c974d9d54cc7e18ad158de5/modules/manage/partials/mountable-topics.adoc?plain=1#L17

We can go ahead and do that @Deflaimun

File: modules/manage/partials/mountable-topics.adoc [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 ====
 
-ifndef::env-cloud[]
 . xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
 . Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
 endif::[]
 ifdef::env-cloud[]
 . xref:manage:rpk/rpk-install.adoc[Install `rpk`] to run cluster mount and unmount operations on the command line, or xref:manage:api/cloud-api-authentication.adoc[authenticate] to the Cloud API and execute these operations programmatically.
 +
 If using the Cloud API, you'll make requests against Data Plane APIs. Make sure you xref:manage:api/cloud-dataplane-api.adoc#get-data-plane-api-url[retrieve the Data Plane API URL] and use this URL to perform a mount or unmount operation.
```
Comment (kbatuigas) [inline modules/manage/partials/mountable-topics.adoc]: Ah, now that we're removing the bullet point regarding Tiered Storage for the cloud version we should change the formatting for the cloud prereqs. I think we could do either an unordered list, but only if "If using the cloud API" is a nested bullet after the main one, or we get rid of the listing altogether. I could go either way.

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Requesting a change to the prereq formatting for Cloud, since we're getting rid of one of the list items.

File: modules/manage/partials/mountable-topics.adoc (hunk 1 above)
Comment (Deflaimun) [reply modules/manage/partials/mountable-topics.adoc]: What about this?
https://github.com/redpanda-data/docs/pull/894/commits/18ffdf964a904041244855f935e9d77bdc275432

File: modules/manage/partials/mountable-topics.adoc [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 
+
+ifndef::env-cloud[]
 [NOTE]
 ====
 include::shared:partial$enterprise-license.adoc[]
 ====
 
-ifndef::env-cloud[]
 . xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
```
Comment (kbatuigas) [inline modules/manage/partials/mountable-topics.adoc]:
```suggestion
* xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
```

File: modules/manage/partials/mountable-topics.adoc [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
 . Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
```
Comment (kbatuigas) [inline modules/manage/partials/mountable-topics.adoc]:
```suggestion
* Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
```

File: modules/manage/partials/mountable-topics.adoc [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
 endif::[]
 ifdef::env-cloud[]
-. xref:manage:rpk/rpk-install.adoc[Install `rpk`] to run cluster mount and unmount operations on the command line, or xref:manage:api/cloud-api-authentication.adoc[authenticate] to the Cloud API and execute these operations programmatically.
-+
-If using the Cloud API, you'll make requests against Data Plane APIs. Make sure you xref:manage:api/cloud-dataplane-api.adoc#get-data-plane-api-url[retrieve the Data Plane API URL] and use this URL to perform a mount or unmount operation.
-. Enable xref:ROOT:manage:tiered-storage.adoc[Tiered Storage] for the topics you want to unmount or mount.
+. xref:manage:rpk/rpk-install.adoc[Install `rpk`] to run cluster mount and unmount operations on the command line or
```
Comment (kbatuigas) [inline modules/manage/partials/mountable-topics.adoc]:
```suggestion
xref:manage:rpk/rpk-install.adoc[Install `rpk`] or xref:manage:api/cloud-api-authentication.adoc[authenticate] to the Cloud API.
```

File: modules/manage/partials/mountable-topics.adoc [hunk 5]
```diff
@@ -1,22 +1,20 @@
...
+. xref:manage:api/cloud-api-authentication.adoc[Authenticate] to the Cloud API and make requests against the Data Plane APIs. xref:manage:api/cloud-dataplane-api.adoc#get-data-plane-api-url[Retrieve the Data Plane API URL] and use it to perform a mount or unmount operation.
```
Comment (kbatuigas) [inline modules/manage/partials/mountable-topics.adoc]:
```suggestion

If using the API, make sure that you have the correct xref:manage:api/cloud-dataplane-api.adoc#get-data-plane-api-url[Data Plane API URL].
```

**Review by kbatuigas — APPROVED** [review]
lgtm, thank you!

**Pull request merged**